### PR TITLE
Unify and cleanup newsflash update and scraping

### LINKED ISSUE #1260: Refactor scraping news flash
The modules handling the scraping of news flash scraping require extensive refactoring.
Here's a non-exhaustive list of quality issues I see:

- [x]  Multiple occurrences of swallowed exceptions - I mean `Exception`s (catching `Exception`, even when logged, seems to be overkill).

- [x] Module-level side effects: `init_flask()`
- [x] Naming: Unclear separation of `news_flash_*` files and a `news_flash` folder. Noninformative long names; "scrap" instead of "scrape" (the correct verb);  `beautiful_soup` as part of a file name. The database adapter is named `news_flash_parser`. Regular links are named `rss_link`.
- [x] `get_*` functions receive site as argument to dispatch completely different handling for each site; I believe this should be "transposed" to separate modules/classes/config for each site (group by site, not by task, since they are not similar). Moreover, it seems easy to create a single function that would return a dict[str, str] which will hide the difference in source from other parts of the parsing.
- [x] `u''` strings
- [x] Different classifier functions for ynet and twitter, using separate lists, hard coded inside the conditionals. The classifier for both walla and ynet is called `classify_ynet`.
- [x] Commented-out imports and other minor style issues such as non-PEP8 compliant spaces
- [x] Using deprecated `pd.np` instead of numpy directly.
- [x] Unnecessarily repeated fetching and parsing of news flashes, just to check if there are new ones.
- [ ] Duplicated logic for location extraction on `update_news_flash`
- [x] Use SQLAlchemy objects (`NewsFlash`) instead of hand crafted SQL.
Of course, to in order to make large changes, we need tests. I guess another issue is in order for that.

### PR DESCRIPTION
**Pull request by elazarg**
* Fix #1260 (last item): use ORM api for the updater. Note that no explicit batching is performed, and it updates everything, regardless of what has changed.
* Fix what seems like a copy-paste bug in scrape_extract_store: classification requires the content, not the title, or at least no only the title. It might require two calls though; feedback is welcome.
* Remove unused (or seemingly-unused) methods in DBAdapter, and change others to use the ORM api.
* Group similar functions together, so they can be compared, maintained consistently, and hopefully refactored to have fewer code duplications.

### CODE REVIEW

File: anyway/parsers/news_flash.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            extract_geo_features(db, newsflash)
+    db.commit()
+
+
+def scrape_extract_store_rss(site_name, db):
+    latest_date = db.get_latest_date_of_source(site_name)
+    for newsflash in rss_sites.scrape(site_name):
+        if newsflash.date <= latest_date:
+            break
+        newsflash.accident = classify_rss(newsflash.description or newsflash.title)
```
Comment (elazarg) [inline anyway/parsers/news_flash.py:40]: Used to be `newsflash.title`. Perhaps this should be two calls.

Comment (elazarg) [reply anyway/parsers/news_flash.py:40]: Perhaps the right thing to do is `'\n'.join([description, title])` either here or inside the classification.

**Comment by elazarg** [comment]
How should I re-trigger the test? It failed for no reason.

**Comment by atalyaalon** [comment]
> How should I re-trigger the test? It failed for no reason.

@idosavion any idea why codecov test is failing?

File: anyway/parsers/news_flash.py (hunk 1 above)
Comment (atalyaalon) [reply anyway/parsers/news_flash.py:40]: @elazarg sounds good, perhaps title and then description?
We need to understand how it influences the extraction of the location string. @Mano3 what do you think?

**Comment by idosavion** [comment]
Seems like the failure to upload is sporadic and everything is fine now, there is a rerun button on the actions tab

**Comment by elazarg** [comment]
Well, re-running doesn't work. I'm not sure what can I do about this failure.

**Comment by idosavion** [comment]
I'll release a fix for this soon, but I think it's just a momentarily timeout on codecov

File: anyway/parsers/news_flash.py (hunk 1 above)
Comment (Mano3) [reply anyway/parsers/news_flash.py:40]: Sorry, due to a bug this was only sent now.

Comment (elazarg) [reply anyway/parsers/news_flash.py:40]: We still don't have a proper validation set on which to work. If you don't consider this as a bug, I'd better leave it the way it used to be.

**Comment by Mano3** [comment]
@idosavion, is it safe to merge this PR?

**Comment by elazarg** [comment]
@Mano3 I've just committed a change, as I understood you wanted me to. Is it correct?

**Pull request merged**